components/VM_Arm/configurations/vm.h

```diff
@@ -68,22 +68,28 @@
     attribute int num_extra_frame_caps; \
     attribute int extra_frame_map_address; \
     attribute { \
-        string linux_ram_base; \
-        string linux_ram_paddr_base; \
-        string linux_ram_size; \
-        string linux_ram_offset; \
+        string ram_base; \
+        string ram_paddr_base; \
+        string ram_size; \
+        string ram_offset; \
         string dtb_addr; \
         string initrd_max_size; \
         string initrd_addr; \
-    } linux_address_config; \
+        string kernel_entry_addr = "0"; \
+    } vm_address_config; \
     attribute { \
-        string linux_name = "linux"; \
+        string kernel_name = "linux"; \
         string dtb_name = "linux-dtb"; \
         string initrd_name = "linux-initrd"; \
-        string linux_bootcmdline = ""; \
-        string linux_stdout = ""; \
+        string kernel_bootcmdline = ""; \
+        string kernel_stdout = ""; \
         string dtb_base_name = ""; \
-    } linux_image_config; \
+        int provide_dtb = true; \
+        int generate_dtb = false; \
+        int provide_initrd = true; \
+        int clean_cache = false; \
+        int map_unity = false; \
```

Comment:
I wonder if "one_to_one" is more intuitive as a name than "unity"?
```suggestion
        int map_one_to_one = false; \
```

Reply:
Done.